tst/scripts/utils/artemis.py

```diff
@@ -122,6 +148,130 @@ def run(nproc, input_filename, arguments, restart=None):
         os.chdir(current_dir)
 
 
+# Function for reading Artemis data
+def load_level(n, base, dir="./", off=0):
```

Comment:
Should we just import `ahdf.py` to do this work? 

I already use `ahistory.py` elsewhere in the tests for analysis.

Reply:
Maybe, this was me just copying my normal loader for uniform / nested refinement meshes. I don't know what all goes on in `ahdf.py` since I don't use it.

Reply:
OK I see now that this was moved, not added, thanks. 

I do think that within the artemis repository we should have a single maintained path for interpreting hdf5 outputs, and that is `ahdf.py` (which is derived from `phdf.py` and in that sense is "parthenonic"). But that is a change for another PR.